### DISCUSSION TITLE
Is there any way to control connection pooling from the graphql rebing library?

### DISCUSSION THREAD

**Discussion by drewgallagher** [rebing/graphql-laravel#1075] [Q&A]
Right now our API being triggered many lambda invocations. 

As a result we are opening up many connections to our RDS database. 

The connections are exceeding our connection limit currently.

Is there a way to manage connection pooling so we do not exceed our maximum connections?

**Comment by mfn** [comment]
That's not related to graphql or this package, you need to use something like https://aws.amazon.com/rds/proxy/
